src/components/Edit/ChangeSummary.vue

```diff
@@ -57,12 +57,18 @@
       <article id="org-person-summary-section" class="section-container">
         <v-row no-gutters>
           <v-col cols="12" sm="3">
-            <label>{{ firmLabel }} Information</label>
+            <label>
+              {{ this.getResource.entityType === 'SP' ? 'Proprietor' : 'Partner' }} Information
+            </label>
```

Comment:
Labels are [inline elements](https://www.w3schools.com/html/html_blocks.asp), so newlines matter.

If this displays OK, leave it for now.

Reply:
> LGTM. Thanks for the small changes!

Thanks for the PR :D